pkg/crc/segment/segment_test.go

```diff
@@ -30,9 +30,10 @@ type segmentResponse struct {
 			ExperimentalFeatures bool   `json:"enable-experimental-features"`
 		} `json:"traits"`
 		Properties struct {
-			Error   string `json:"error"`
-			Version string `json:"version"`
-			CPUs    int    `json:"cpus"`
+			Error     string `json:"error"`
+			ErrorType string `json:"error-type"`
```

Comment:
Wondering if in the long run we could send only `ErrorType` and not the error message. To be useful, this implies fine-grained error types rather than always returning `fmt.Errorf`, but this would remove the risk that the error message contains personal user information.